### PR TITLE
chore: update pr template

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## What are the changes introduced in this PR?

Updating the pull request template.

## Please explain the objectives of your changes below

Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here

### Type of change

If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please fill in the details on the changes made.

- Existing capabilities/behavior

- New capabilities/behavior

If the pull request is a **new feature**,

### Any technical or performance related pointers to consider with the change?

N/A

### Any new dependencies introduced with this change?

N/A

### Any new generic utility introduced or modified. Please explain the changes.

N/A

### If the PR has changes in more than 10 files, please mention why the changes were not split into multiple PRs.

N/A

### If multiple linear tasks are associated with the PR changes, please elaborate on the reason:

N/A

<hr>

### Developer checklist

- [ ] **No breaking changes are being introduced.**

- [ ] Are all related docs linked with the PR?

- [ ] Are all changes manually tested?

- [ ] Does this change require any documentation changes?

- [ ] Are relevant unit and component test-cases added?

### Reviewer checklist

- [ ] Is the type of change in the PR title appropriate as per the changes?

- [ ] Verified that there are no credentials or confidential data exposed with the changes.

### CODE REVIEW

**Review by koladilip — DISMISSED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Write a brief explainer on your code changes.
 
+## What is the related Linear task?
+
+Link to linear ticket.
```
Comment (krishna2020) [inline .github/pull_request_template.md:7]:
```suggestion
Resolves INT-<>
```

Comment (krishna2020) [reply .github/pull_request_template.md:7]: mentioning complete linear ticket will expose the url

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -14,8 +18,6 @@ If the pull request is a **bug-fix**, **enhancement** or a **refactor**, please
 
 - New capabilities/behavior
 
-If the pull request is a **new feature**,
-
 ### Any technical or performance related pointers to consider with the change?
```
Comment (krishna2020) [inline .github/pull_request_template.md:21]: we need to keep relevant questions to make it user friendly. Can we move queries on 10 files & multiple linear tasks as Developer checklist instead ?
- Limited to 10 file changes in PR
- Limited to one linear task 

As these things shouldn't be the norm, we can have discussion as comments instead if this has to be exception. cc: @koladilip

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 Write a brief explainer on your code changes.
 
+## What is the related Linear task?
+
+Link to linear ticket.
+
 ## Please explain the objectives of your changes below
 
 Put down any required details on the broader aspect of your changes. If there are any dependent changes, **mandatorily** mention them here
```
Comment (krishna2020) [inline .github/pull_request_template.md:11]: There's too much of content under `Type of change`. Remove all that & convert to question in bold:
### Any changes to existing capabilities/behaviour, mention the reason & what are the changes ?

File: .github/pull_request_template.md (hunk 2 above)
Comment (krishna2020) [inline .github/pull_request_template.md:21]: Move this question to 4th position, should be last one

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -42,11 +32,15 @@ N/A
...
 
-- [ ] Are all related docs linked with the PR?
+- [ ] All related docs linked with the PR?
+
+- [ ] All changes manually tested?
+
+- [ ] Any documentation changes needed with this change?
 
-- [ ] Are all changes manually tested?
+- [ ] Are we limited to 10 file changes in the PR?
```
Comment (krishna2020) [inline .github/pull_request_template.md:41]:
```suggestion
- [ ] Is the PR limited to 10 file changes?
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -42,11 +32,15 @@ N/A
...
 
-- [ ] Does this change require any documentation changes?
+- [ ] Are we limited to one linear task?
```
Comment (krishna2020) [inline .github/pull_request_template.md:43]:
```suggestion
- [ ] Is the PR limited to one linear task?
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -42,11 +32,15 @@ N/A
```
Comment (krishna2020) [inline .github/pull_request_template.md:32]:
```suggestion
- [ ] My code follows the style guidelines of this project

```

**Review by krishna2020 — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Pull request merged**